### DISCUSSION TITLE
Interface localization

### DISCUSSION THREAD

**Discussion by Akimotorakiyu** [rust-lang/crates.io#4333] [Feature Requests]
Interface localization

**Comment by kzys** [comment]
Can we use https://github.com/ember-intl/ember-intl ?

**Comment by fbstj** [comment]
@Manishearth thoughts on adding crates.io to the l10n efforts (and adding it to Pontoon)?

**Comment by Manishearth** [comment]
Go for it, if y'all can get Fluent integration the same way Servo did, I can hook this up to Pontoon.

**Comment by Manishearth** [comment]
(My "go for it" is not an endorsement of this idea from the rust team, just personally saying that this is a good idea: you should probably work with the crates.io team to ensure it's done right)

**Comment by Akimotorakiyu** [comment]
Please have look at MDN(mozilla developer docs). 

It provides an edit button and a page for editing and translating documents. It can be logged in with a GitHub account, allowing users to translate part of the article for contributions, which makes large-scale collaborative translation possible.

### 1. doc page
![图片](https://user-images.githubusercontent.com/31165554/60475261-12c7c980-9ca9-11e9-9b1e-629aa087a4b2.png)
### 2. edit page
![图片](https://user-images.githubusercontent.com/31165554/60475336-72be7000-9ca9-11e9-937b-efb4d9476cf6.png)

**Comment by sgrif** [comment]
@SilverLeaves While that's nice in theory, building a system like that isn't something that we have the capacity for right now.
